internal/broker/bind_delete.go

```diff
@@ -35,6 +37,12 @@ func (b *UnbindEndpoint) Unbind(ctx context.Context, instanceID, bindingID strin
 	instance, err := b.instancesStorage.GetByID(instanceID)
 	switch {
 	case dberr.IsNotFound(err):
+		// instance does not exist in instance table, but binding could be still present - clean and return 410 Gone
```

Comment:
Can we move this check to condition with `if lastOperation.Type != internal.OperationTypeDeprovision` just to have one `bindingsStorage.Delete` call in the method?

Reply:
Removed - this was quotation from the ticket.